vllm/model_executor/models/llava.py

```diff
@@ -172,7 +174,7 @@ def forward(
                 image_features = image_input
             vision_embeddings = self.multi_modal_projector(image_features)
             inputs_embeds = self.language_model.get_input_embeddings(input_ids)
-            _merge_vision_embeddings(
+            inputs_embeds = _merge_vision_embeddings(
```

Comment:
Is this a bug?

Reply:
No, it is just to make explicit the fact that `inputs_embeds` is modified.